rate.go

```diff
@@ -78,7 +78,7 @@ func (r *rateLimitResult) RetryAfter() time.Duration { return r.retryAfter }
 // Rate describes a frequency of an activity such as the number of requests
 // allowed per minute.
 type Rate struct {
-	period time.Duration // Time between equally spaced requests at the rate
+	Period time.Duration // Time between equally spaced requests at the rate
```

Comment:
Now that this is a public field, could you move the comment up to appear above it and change it to follow go convention to read something like "Period is the amount of time between equally spaced requests for this rate."

Reply:
Oh, thanks, I didn't realize side comments were treated differently.

Reply:
I'm not sure they are, but the important part is to have the documentation start with "Period ..." because that's a Go convention for public field documentation that golint will complain about if not followed. (Moving the comment above just gives us a little more space.)